implementations/rust/ockam/ockam_node/src/api.rs

```diff
@@ -1,10 +1,34 @@
+#![allow(missing_docs)]
+
 use crate::Context;
+use alloc::collections::btree_map::Entry;
+use cddl_cat::{context::BasicContext, flatten::flatten, parse_cddl};
 use core::fmt::Display;
 use minicbor::Encode;
 use ockam_core::api::{assert_request_match, RequestBuilder};
 use ockam_core::compat::vec::Vec;
 use ockam_core::{Address, LocalInfo, Result, Route};
```

Comment:
> There are a lot of unwrap() calls at the moment. I am not really familiar with ockam error handling. Where should I propagate these possible errors to?

There is a central `Error` type that you can use at `ockam_core`. In this case, you can just import that type and use it from both `ockam_core::api` and `ockam_node::api`, as you are already using the `ockam_core::Result`.